### DISCUSSION TITLE
Scaleway integration

### DISCUSSION THREAD

**Discussion by quantumsheep** [Infisical/infisical#5248] [Feature Request]
### Feature description
Be able to synchronize secrets between infisical and Scaleway's infrastructure (Instances, Containers, Elastic Metal, Secret Manager, etc..)

### Why would it be useful?
Scaleway is an ever growing cloud provider used by lots of companies and people. The infisical integration will certainly be used by Scaleway users.

### Additional context
Website: https://scaleway.com
[Community Tools](https://developers.scaleway.com/en/community-tools/) page.

**Comment by dangtony98** [comment]
@quantumsheep  Added to our [roadmap](https://www.notion.so/infisical/be2d2585a6694e40889b03aef96ea36b?v=5b19a8127d1a4060b54769567a8785fa&p=5fe284ae12db44fe83922d09ba885a9d&pm=s)!

We'll be adding these one at a time — Should be pretty quick though, feel free to contribute!

**Comment by chisom5** [comment]
@dangtony98 I would love to take this

**Comment by dangtony98** [comment]
PR #783 was started to address this issue; I'll take it from here and hopefully have it done in the next week.
